.docs/user-guide/storage-providers.md

```diff
@@ -656,14 +656,53 @@ libstorage:
           storagePoolName: storagePoolName
 ```
 
+## DigitalOcean
+Thanks to the efforts of our tremendous community, libStorage also has built-in
+support for DigitalOcean!
+
+<a class="headerlink hiddenanchor" name="digitalocean-block-storage"></a>
+<a class="headerlink hiddenanchor" name="dobs"></a>
+
+### DO Block Storage
+The DigitalOcean Block Storage (DOBS) driver registers a driver named `dobs`
+with the libStorage service registry and is used to attach and mount
+DigitalOcean block storage devices to DigitalOcean instances.
+
+!!! note
+    The DigitalOcean Block Storage driver currently only supports operating in
+    _local only_ mode where the libStorage server must be running on the same
+    host as the client.
+
+#### Requirements
+The DigitalOcean block storage driver has the following requirements:
+
+* Valid DigitalOcean account
+
+#### Configuration
+The following example illustrates a minimal configuration for the DigitalOcean
+block storage driver:
+
+```yaml
+dobs:
+  token: 123456
+  region: us-east-1
```

Comment:
I'd prefer to have this be set to a real region that supports Volumes. Either `nyc1` or `sfo2`.